### ISSUE TITLE
Frequency and hex data display

### ISSUE THREAD

**Issue by PlasmaHH** [merbanan/rtl_433#2260]
Originally I wanted to ask here about the possibility to receive multiple frequencies at once within a given bandwidth, as this is theoretically possible with sdr devices. It seems to be the case, though I can't find it explicitly mentioned in the documentation, I tried receiving at 433.88 and 433.92 and got multiple packets.

This lead me to the frequency output not really being accurate enough. When already the input frequency is two digits and its often not exactly the same at receiving, having .2f instead of .1f output format was beneficial to me to see what is going on.

Then I discovered the http server. This is some really great stuff and should really have a "kitchen sink screenshot" showing off on the front page of this project.

Anyways my real goal was to identify which sensor is which (there are quite some around here) and I noticed that the humidity was not displayed correctly. I could not find an option to just display the package in hex along with the decoded output. Only some really nice and in-depth pulse analysis, which is a bit too detailed, I need something in between, like just some hex output of the packet for the kv output style or an additional optional column for the http interface ( extra points for marking in colours which values are decoded from which hex bytes )

## tl;dr

* .2f instead of .1f output for kv output of frequency ( due to changing frequency and rounding errors, 433.88 and 433.92 would both be displayed as 433.9 but it would be intresting to distinguish them)
* instead of full debug info in the kv output format just output the hex data
* add an optional column in the http output that contains the hex data of a packet ( extremely useful would be coloured backgrounds for which bytes decode to which field values )

**Comment by gdt** [comment]
A lot has changed and it might be good to retest with current git master and file separate feature requests.   I am unclear on whether what you report is about a specific decoder or is more general.

@PlasmaHH are you interested in working on PRs for these changes?   Really things will change if someone submits a well-considered high-quality PR.
